### PR TITLE
feat: impl MultiInstanceProto

### PR DESCRIPTION
**Pull request by killagu**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

**Review by fengmk2 — APPROVED** [review]
+1

File: README.md [hunk 1]
```diff
@@ -156,6 +156,172 @@ export class HelloService {
...
+    name: 'dynamicLogger',
+  })
+  // 通过自定义注解来指定 Qualifier
+  @LogPath('foo')
+  fooDynamicLogger: DynamicLogger;
+
+  @Inject({
+    name: 'dynamicLogger',
+  })
+  @LogPath('bar')
```
Comment (fengmk2) [inline README.md:313]: 不加 Qualifier 默认返回的是第一个？

Comment (killagu) [reply README.md:313]: 必须加。不加会挂的。

Comment (fengmk2) [reply README.md:313]: 好吧，那我将 oss 的写法改一下，默认就是多实例的。

**Pull request merged**